### DISCUSSION TITLE
an improvement to the embeds

### DISCUSSION THREAD

**Discussion by ghost** [discordjs/discord.js#5894] [Q&A] [answered]
Hello.

I was programming and this occurred to me

[Image](https://i.ibb.co/NZff1ZS/a.png)

it would be nice if they changed the `.setColor(<Color>);`

by `.setLine(<line>, [<color>]);`

**Comment by vladfrangu** [comment, marked as answer]
We appreciate the hype, but we are just a library around Discord's API. For feedback, you should use their [feedback website](https://feedback.discord.com) instead!
